### DISCUSSION TITLE
margin in the generated files

### DISCUSSION THREAD

**Discussion by sblum51** [karttapullautin/karttapullautin#97] [Q&A] [answered]
Hello,
I generated several png using the script but for each file I have a margin all around the picture.
Is it an issue in the data ?

![1](https://github.com/rphlo/rusty-pullauta/assets/2675497/1801749c-3d27-485e-9113-76cb102f2860)

**Comment by stefankinell** [comment, marked as answer]
It looks like you are processing one laz-file at a time? Then you get that kind of border for the vegetation. If you put several adjacent laz-files in the same in-folder it should eliminate those blank borders - in the direction where there is a bordering laz-file so to say.

**Reply by sblum51** [reply]
thanks for your reply.
Yes I'm processing one laz-file at a time
For the moment (beta test) the national organization only provides  "*.copc.laz" files for all the country.
@rphlo suggests me to use las2txt and generate a .xyz file (from the .copc.laz) - and I use the xyz file as input for rusty I'm looking.
The batch mode only works with laz files ? 

The current process  :
copc.laz ==[las2txt]=> xyz ==[rusty]==> png

**Reply by stefankinell** [reply]
Ah, yea I saw that conversation. Honestly, I do not know if the batch process can only take .las/.laz-files. Try it out. If it can cope with xyz-files one by one, perhaps it works in bath as well.

**Reply by sblum51** [reply]
I can also try to re-convert txt to .las and use the batch mode

**Reply by sblum51** [reply]
Ok, I have I think I have an issue. 
I can generate png using rusty and "xyz" but when I try to use las or laz the script is blocked on "Identifying knolls".
Per example I converted the xyz to las and send it to rusty it was stucked. When I try directly with "xyz" the generaton took few seconds. 

I think something is missing in our las laz file (or in options)

**Reply by rphlo** [reply]
Hi, the program doesnt take xyz files as inputs in batch mode as they lack headers that considerably ease the batch job.
@guizmo51 you mentionned in another issue that you solved the xyz2las convertion, is this issue still valid?

**Reply by sblum51** [reply]
Hey, 
We can close this issue, I succeeded to generate las file for use with the batch mode.
